packages/ui-default/components/contestProblemEditor/ContestProblemEditor.tsx

```
@@ -0,0 +1,200 @@
+        // pid maybe not exist
+      });
+  }, 500);
+
+  React.useEffect(() => {
+    fetchProblemTitles(problems.map((i) => i.pid).filter((i) => i));
+  }, []);
+
+  const onChange = (newProblems: Problem[]) => {
+    const fixedProblems = newProblems.map((i) => {
+      const problem = { ...i };
+      if (problem.title === '') delete problem.title;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)